web/vtadmin/src/components/routes/Workflows.tsx

```diff
@@ -70,6 +72,12 @@ export const Workflows = () => {
                 <tr key={idx}>
                     <DataCell>
                         <div className="font-bold">{href ? <Link to={href}>{row.name}</Link> : row.name}</div>
+                        <div className="text-secondary text-success-200">
+                            {row.workflowType}
+                            <span className="text-sm">
+                                {row.workflowSubType !== 'None' ? ' (' + row.workflowSubType + ')' : ''}
+                            </span>
```

Comment:
Here you could conditionally render the entire `span` if the workflow subtype is 'none;
```suggestion
                            {row.workflowSubtype && row.workflowSubtype != 'None' && (
                                <span className="text-sm">
                                    {' (' + row.workflowSubType + ')'}
                                </span>
                            )}
```

Reply:
It also takes care of the nil check for `row.workflowSubType`!